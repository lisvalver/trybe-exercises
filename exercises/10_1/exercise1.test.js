const sumNumbers = require('./exercise1');

