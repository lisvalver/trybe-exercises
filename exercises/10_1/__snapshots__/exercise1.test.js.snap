// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste se a função sum lança um erro quando os parametros são 4 e "5"(string 5) 1`] = `"parameters must be numbers"`;

exports[`Teste se a função sum lança um erro quando os parametros são 4 e "5"(string 5): parameters must be numbers 1`] = `"parameters must be numbers"`;

exports[`Teste se a mensagem de erro é "parameters must be numbers" quando realizar a chamada sum(4, "5"): parameters must be numbers 1`] = `"parameters must be numbers"`;
